src/components/DeleteHelicopter.jsx

```diff
@@ -0,0 +1,87 @@
+import { useEffect } from "react";
+import { useDispatch, useSelector } from "react-redux";
+import {
+  fetchHelicopters,
+  deleteHelicopter,
+  updateHelicopters,
+} from "../features/helicopters/helicopterSlice";
+import LayoutComponent from "../Layout";
+import { Button, Table, Popconfirm } from 'antd';
+import { QuestionCircleOutlined } from "@ant-design/icons";
+
+const DeleteHelicopterComponent = () => {
+  //hooks
```

Comment:
- [ ] The readme still has wrong instructions:

![image](https://github.com/emmanuelsteven/heli-hire-captsone-frontend/assets/57421823/d8cead0c-c2b1-4c1d-9fe5-a0c0174d0100)